Gemfile

```diff
@@ -100,6 +100,7 @@ group :development, :test do
   gem 'rspec-benchmark'
   gem 'parallel_tests'
 	gem 'bullet'
+  gem 'bundler', '1.17.1'
```

Comment:
Can you post some of what led you to add this?

Reply:
Sure! So when I would try to bundle I kept getting errors that essentially said Bundler was v 2.2-something, and that messed with some dependencies. When I moved to an earlier version of Bundler, it let me `bundle install` without issues. My thought process was that for the older gems, maybe specifying a Bundler version would be helpful? Honestly picked one at random, as the messaging I got said anything lower than (I wanna say) 2.0.

Reply:
Hmm, I am not sure what this is really doing for us within just the dev and test block as I think usually we run `bundle install` essentially from root.